install_config/install/prerequisites.adoc

```diff
@@ -362,6 +362,124 @@ SELINUX=enforcing
 SELINUXTYPE=targeted
 ----
 
+[[prereq-cloud-provider-considerations]]
+
+=== Cloud Provider Considerations
+
+*Set up the Security Group*
+
+When installing on AWS or OpenStack, ensure that you set up the security group.
+
+[cols="1,2"]
+|===
+|*All OpenShift Hosts*
+a|tcp/22 from host running the installer/Ansible
```

Comment:
Eh, I take it back. I think we've previously shied from calling it "the installer", but that's what's in the `atomic-openshift-installer` command name and `installer.cfg.yml` file name, for example, and that's what people have generally called it, so I don't know why.